sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -11,8 +11,10 @@ import { Delivery } from 'rhea-promise';
 import { HttpOperationResponse } from '@azure/core-http';
 import Long from 'long';
 import { MessagingError } from '@azure/core-amqp';
+import { OperationOptions } from '@azure/core-auth';
 import { RetryOptions } from '@azure/core-amqp';
 import { TokenCredential } from '@azure/core-amqp';
+import { TokenCredential as TokenCredential_2 } from '@azure/core-auth';
```

Comment:
`TokenCredential` is being imported twice. Why?

Reply:
Probably an auto-resolve-import gone awry. I'll find it.